### DISCUSSION TITLE
how to disable edgemesh?

### DISCUSSION THREAD

**Discussion by caelumpirata** [kubeedge/kubeedge#4781] [Q&A] [answered]
i'm following this guide and wanted to change the value of edgemesh enabled false (error handling section from below link)
[https://www.kubesphere.io/zh/blogs/kubesphere-kubeedge-edgemesh/#%E7%AE%80%E4%BB%8B](url)

vim /etc/kubeedge/config/edgecore.yaml
this module is missing in the edgecore.yaml file. How can i edit it now?
`modules:
  ..
  edgeMesh:
    enable: false`

**Comment by vincentgoat** [comment, marked as answer]
Edgemesh is now a network plugin and not a built-in module of KubeEdge, so it is no need to disable it in the edgecore.yaml file.

**Reply by vincentgoat** [reply]
FYI: https://edgemesh.netlify.app/

**Reply by caelumpirata** [reply]
while helm installing edgemesh, doesn't it create **edgemesh-server** pod? 
I got my edgemesh-agent ready on my edgenode and master node both. but still can't access my application deployed on edge node through load balancer, but with nodeport i can!

**Reply by vincentgoat** [reply]
@Poorunga

**Reply by caelumpirata** [reply]
is there any updated link for the EdgeNode connection using edgemesh in kubesphere ?

**Reply by vincentgoat** [reply]
Suggest finding a guide link on the Kubesphere website.
